eng/emsdk.proj

```diff
@@ -106,6 +106,32 @@
     <Delete Files="$(ArtifactsObjDir)upstream\emscripten\third_party\*.*" />
     <!--<Delete Files="$(ArtifactsObjDir)upstream\bin\wasm-ld.exe" />--> <!-- TODO: this is used by emcc but could be a symlink to ld.exe -->
 
+    <ItemGroup Condition="'$(PackageRID)' == 'win-x64'">
```

Comment:
I'd prefer deleting the files on all platforms so we catch potential issues earlier given most of us use mac/linux.